assets/stylesheets/_mixins.scss

```diff
@@ -377,6 +377,8 @@
 	select {
 		font-size: $default-font-size;
 		color: $dark-gray-500;
+		height: auto;
```

Comment:
A quick note about the changes in this file: they mostly just override styles inherited from WP-Admin. We may actually want to fix these issues in WP-Admin first, rather than overwrite them here.

Reply:
Also, it turns out `min-height` does not work in Safari for some reason. Which is less than great. I found [some potential overrides](https://stackoverflow.com/questions/13252835/safari-change-select-height), but none of them appear to work in the newer versions of Safari.